### PR TITLE
Add chart for kube-ebs-snapshots

### PR DESCRIPTION
**Pull request by goruha**
## What
* Create chart to make ebs snapshots

## Why
* Ebs snapshots is a good way to create backups

### CODE REVIEW

File: incubator/kube-ebs-snapshots/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
+1) You need permissions to allow backup write data into s3 bucket.
```
Comment (osterman) [inline incubator/kube-ebs-snapshots/templates/NOTES.txt]: Use terraform comments to facilitate copy & paste

Comment (goruha) [inline incubator/kube-ebs-snapshots/templates/NOTES.txt]: Done

**Comment by osterman** [comment]
> @TODO: Uncomment it when helm version > 2.3.1 would be used

Let's just require helm > 2.3.1

**Comment by goruha** [comment]
Close for now
Reopen in future if needed

**Pull request closed without merging**